news/2017-10-11-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "The QAT Gazette: October 2017"
+date: 2017-10-11 09:00:00 +0000
+---
+
+There’s been major developments in the Quality Assurance Team and Beatmap Nominators this month, with new QAT leaders, changes to the Nominator Tier system, revival of mapping contests, and more ranked maps per day! Curious to know more? Read on to find out!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+Hello again everyone! Another month has passed, so here we are with the second issue of the **QAT Gazette**!
+This month was full of events going on in the Quality Assurance Team, the Beatmap Nominators, and of course osu! itself. New Beatmap Nominators have been introduced for osu!taiko, osu!catch and osu!mania, the QAT and Beatmap Nominator Upheaval and the return of Mapping with Rewards are just some of the few topics in this month’s issue, so let’s begin!
+
+## The Upheaval
+
+The Upheaval was probably the biggest event of changes affecting the Quality Assurance Team and the Beatmap Nominators we have seen for a long time. For these who may have missed the details, you can still read more about it in the [Development Forum Thread](https://osu.ppy.sh/community/forums/topics/635507)!
+
+### No More Tiers
+
+As we discussed last month, the biggest point in the Upheaval was **the removal of the Beatmap Nominator Tier system**. As you may know, we previously split the osu! Beatmap Nominators into two groups - Tier 1 and Tier 2. Tier 1 members were able to make the first nomination for maps only, whereas Tier 2 members were able to nominate and also qualify beatmaps. 
```

Comment:
either change qualify for second nomination or change all nominations for bubbles, otherwise it becomes confusing